doc/Readme.md

```diff
@@ -42,6 +43,43 @@ sudo ./opentap.linux --quiet
 The installer is likely to work on other Linux distributions, but additional dependencies
 may be required on these platforms, such as dotnet 6 runtime.
 
+#### Other Linux distributions
+For other Linux distributions, OpenTAP can be installed using the terminal:
+
+1. Install dotnet: [Instructions](https://learn.microsoft.com/da-dk/dotnet/core/install/linux?WT.mc_id=dotnet-35129-website).
+    - For Ubuntu: `sudo apt install -y dotnet-sdk-8.0`
+2. Download the latest OpenTAP release: `curl -Lo opentap.zip 'https://packages.opentap.io/4.0/Objects/Packages/OpenTAP?os=Linux&architecture=<architecture>'`
+    - Remember to change the architecture if needed (`x64`, `arm64`, `arm`).
+3. Create a directory for OpenTAP: `sudo mkdir /opt/opentap && sudo chown $USER:$USER /opt/opentap`
+4. Extract the OpenTAP release: `unzip ./opentap.zip -d /opt/opentap`
+    - If you don't have unzip installed, you can install it with `sudo apt install unzip`.
+5. Make tap executable: `chmod +x /opt/opentap/tap`
+6. Create a symbolic link to tap: `sudo ln -s /opt/opentap/tap /usr/local/bin/tap`
+
+##### An example using Ubuntu 24.04
+This example shows how to install OpenTAP on ARM64 Ubuntu 24.04 running on a Raspberry PI 5. The steps are similar for other distributions.
+
+```bash
+# Install dotnet and unzip
+sudo apt install -y dotnet-sdk-8.0 unzip
+
+# Download the latest OpenTAP release
+curl -Lo opentap.zip 'https://packages.opentap.io/4.0/Objects/Packages/OpenTAP?os=Linux&architecture=arm64'
```

Comment:
Our example should probably be for amd64 instead of arm64 since it is much more common